root/usr/lib64/ruby/vendor_ruby/discovery/screen/network.rb

```diff
@@ -30,7 +30,7 @@ def screen_network mac, ip = cmdline('fdi.pxip', ''), gw = cmdline('fdi.pxgw', '
       return [:screen_network, mac, ip, gw, dns]
     end
     action = Proc.new { configure_network true, mac, ip, gw, dns }
-    [:screen_info, action, "Configuring network via DHCP. This operation can take several minutes to complete.", "Unable to bring network via DHCP",
+    [:screen_info, action, "Configuring network. This operation can take several minutes to complete.", "Unable to bring network via DHCP",
```

Comment:
What about the second error message?  "Unable to bring network via DHCP" - could you hit that in static network config?